rules/windows/process_creation/proc_creation_win_bash_command_execution.yml

```diff
@@ -0,0 +1,27 @@
+title: Indirect Inline Command Execution Via Bash.EXE
+id: 5edc2273-c26f-406c-83f3-f4d948e740dd
+status: experimental
+description: Detects execution of Microsoft bash launcher with the "-c" flag. This can be used to potentially bypass defenses and execute linux or windows based binaries directly via bash
```

Comment:
```suggestion
description: Detects execution of Microsoft bash launcher with the "-c" flag. This can be used to potentially bypass defenses and execute Linux or Windows-based binaries directly via bash
```